INSTALL.md

```diff
@@ -0,0 +1,38 @@
+#If you are in a hurry
+
+On Linux (Debian):
+
+```sh
+ $ [sudo] apt-get install libgrpc-dev
+```
+
+On MacOS:
+
+```sh
+ $ curl -fsSL https://goo.gl/getgrpc | bash -
```

Comment:
If you follow that link, it just goes to the latest version of https://github.com/grpc/homebrew-grpc/blob/master/scripts/install